### DISCUSSION TITLE
There is currently an issue with the installation of eww

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#69] [Announcements]
Dear ML4W users,

there is currently an issue with the installation of eww. eww is needed to run the ML4W Sidebar. I have already opened an issue on https://github.com/elkowar/eww/issues/1141 and waiting for feedback.

You can also follow the development here: https://github.com/mylinuxforwork/dotfiles/issues/68

The rest of the ML4W Dotfiles should work fine. You have access to the ML4W Apps from the App Starter.

**Comment by mylinuxforwork** [comment]
I have replaced eww with ags in the latest rolling release. The new version 2.9.4 will be released very soon.

**Comment by PSGitHubUser1** [comment]
Yeah, the `ewww` is not installing w/ `yay`

**Comment by mylinuxforwork** [comment]
I have just launched the new version ML4W Dotfiles 2.9.4 with a new ags based Sidebar. eww is not needed anymore.

**Comment by ErwinG75** [comment]
Uhh I think there's an issue with 2.9.6 (the one from your latest vid). I can open everything except sidebar. No matter how many times I click it, it just doesn't do anything

**Reply by mylinuxforwork** [reply]
This is not eww anymore. The sidebar is based on ags. Can you just type ags into your terminal andcheck if rhere are any error messages.

**Reply by ErwinG75** [reply]
Ah I know this is not Eww anymore, I just don't know where to tell you because this is my first time actually using Github (outside of downloading/git) 😅

There are no error message, it just shows this

![image](https://github.com/user-attachments/assets/ee519f41-59b9-4e45-992e-164345591df7)
